experimental/apps-mcp/cmd/install.go

```diff
@@ -18,14 +29,20 @@ func newInstallCmd() *cobra.Command {
 		Short: "Install the Apps MCP server in coding agents",
 		Long:  `Install the Databricks Apps MCP server in coding agents like Claude Code and Cursor.`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return runInstall(cmd.Context())
+			return runInstall(cmd)
 		},
 	}
 
+	cmd.Flags().StringP("profile", "p", "", "~/.databrickscfg profile")
+	cmd.RegisterFlagCompletionFunc("profile", profile.ProfileCompletion)
```

Comment:
I wouldn't add a flag for this; no one will use it, and we may not want to keep this way of doing authentication in future versions.